### PR TITLE
Demos update

### PR DESCRIPTION
**Pull request by jedi1150**
- Dependencies updated to the latest versions
- Control buttons state optimised in all demos
- `customVideoSource` now releases on call end

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -2,29 +2,29 @@
...
         core_ctx_version = '1.6.0'
-        constraintLayout_version = '2.0.4'
+        constraintLayout_version = '2.1.1'
         vectordrawable_version = '1.1.0'
         preferences_ktx_version = '1.1.1'
         lifecycle_ktx_version = '2.3.1'
-        activity_ktx_version = '1.2.3'
-        cameraX_version = '1.1.0-alpha05'
+        activity_ktx_version = '1.3.1'
+        cameraX_version = '1.1.0-alpha08'
```
Comment (YuliaGrigorieva) [inline build.gradle:21]: 1.1.0-alpha09 version is available, please check if we can update to it

File: videocall-deepar/src/main/java/com/voximplant/demos/kotlin/videocall_deepar/stories/call/CallViewModel.kt [hunk 2]
```diff
@@ -80,6 +80,8 @@ class CallViewModel : BaseViewModel() {
             enableVideoButton.postValue(true)
```
Comment (YuliaGrigorieva) [inline videocall-deepar/src/main/java/com/voximplant/demos/kotlin/videocall_deepar/stories/call/CallViewModel.kt:80]: I think we need to check if the call is on hold here

Comment (jedi1150) [reply videocall-deepar/src/main/java/com/voximplant/demos/kotlin/videocall_deepar/stories/call/CallViewModel.kt:80]: there is no hold function in this demo

**Review by YuliaGrigorieva — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (jedi1150) [reply build.gradle:21]: Upgrading to 1.1.0-alpha09 requires some additional code changes. The version will be updated later.

**Pull request merged**